src/briefcase/commands/base.py

```diff
@@ -687,6 +720,14 @@ def add_default_options(self, parser):
 
         :param parser: a stub argparse parser for the command.
         """
+        parser.add_argument(
+            "-C",
+            "--config",
+            dest="config_overrides",
+            action="append",
+            metavar="KEY=VALUE",
+            help="Override the value of the app configuration item KEY with VALUE.",
```

Comment:
nitpick: no period at the end to match the other options (although, I also shouldn't have added one to `-v`...)
```suggestion
            help="Override the value of the app configuration item KEY with VALUE",
```

Reply:
Interesting - the very next option defined (`--verbosity`) *did* have a period. I've corrected both.